.github/workflows/ci.yml

```diff
@@ -165,7 +181,27 @@ jobs:
         sudo ./ci/install_bazelisk.sh
     - name: run tests
       run: ./ci/do_ci.sh bazel.test
-
+  bazel_deprecated_metrics_test:
```

Comment:
nit - add newline before and after this test.

Reply:
thanks, done